Source/DafnyLanguageServer/Language/SyntaxTreeVisitor.cs

```diff
@@ -445,9 +452,21 @@ public abstract class SyntaxTreeVisitor {
         case ITEExpr ifThenElseExpression:
           Visit(ifThenElseExpression);
           break;
+        case LambdaExpr lambdaExpr:
+          Visit(lambdaExpr);
+          break;
         case ForallExpr forAllExpression:
           Visit(forAllExpression);
           break;
+        case ExistsExpr existsExpr:
```

Comment:
So far, the language server avoids the use of abbreviations. I'd prefer to keep this style.